src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoPercentNegativePattern.cs

```diff
@@ -12,21 +12,30 @@ public class NumberFormatInfoPercentNegativePattern
     {
         public static IEnumerable<object[]> PercentNegativePattern_TestData()
         {
-            yield return new object[] { NumberFormatInfo.InvariantInfo, 0, 0 };
-            yield return new object[] { new CultureInfo("en-US").NumberFormat, 0, 1 };
-            yield return new object[] { new CultureInfo("en-MY").NumberFormat, 1, 1 };
-            yield return new object[] { new CultureInfo("tr").NumberFormat, 2, 2 };
+            yield return new object[] { new CultureInfo("en-US").NumberFormat, 1 };
+            yield return new object[] { new CultureInfo("en-MY").NumberFormat, 1 };
+            yield return new object[] { new CultureInfo("tr").NumberFormat, 2 };
         }
 
+        /// <summary>
+        /// Not testing for Windows as the culture data can change
+        /// https://blogs.msdn.microsoft.com/shawnste/2005/04/05/culture-data-shouldnt-be-considered-stable-except-for-invariant/
```

Comment:
I would suggest changing/add the comment telling how we are testing the same thing for Windows in the CultureAllTest.